pelutils/ticktock.py

```diff
@@ -7,13 +7,29 @@
 from pelutils.format import Table
 
 
-class TimeUnit:
-    nanosecond  = ("ns",  1e9)
-    microsecond = ("us",  1e6)
-    millisecond = ("ms",  1e3)
+class TimeUnits:
+    """ Enum-like list of out-of-the-box available units. Format: (suffix, length) """
+    nanosecond  = ("ns",  1e-9)
+    microsecond = ("us",  1e-6)
+    millisecond = ("ms",  1e-3)
     second      = ("s",   1)
-    minute      = ("min", 1/60)
-    hour        = ("h",   1/3600)
+    minute      = ("min", 60)
+    hour        = ("h",   3600)
+
+    @classmethod
+    def units(cls) -> list[tuple[str, float]]:
+        """ List all time units """
+        return [unit for name, unit in cls.__dict__.items() if not callable(getattr(cls, name)) and not name.startswith("__")]
```

Comment:
omfg :P Dine grineren havks kan ikke bare klares med `[k.value[1] for k in cls]`, hvis du lige nedarver fra Enum (https://stackoverflow.com/a/54831960) - men det ville måske ikke tillade dig at have alle dine klassemetoder (der også kunne være modulfunktioner, der bare brugte `TimeUnits`)?

Reply:
TimeUnits er tiltænkt mere som en ikke-udfyldende enum, hvorfor jeg ikke har brugt Enum. Så slipper man også for det der bøvl med at skrive .value hele tiden, og det er nemt bare at bruge sine egne, hjemmebryggede værdier. I princippet kunne det måske også være en namedtuple

Reply:
Finnnnnoooooo.
Den der metode skreg bare enum-opførsel til mig :*